src/lib/apiFavoritesService.js

```diff
@@ -0,0 +1,24 @@
+import Promise from 'bluebird';
+
+import {
+  postWithAuth,
+  getWithAuth,
+  deleteWithAuth,
+} from 'lib/apiService';
+
+import {
+  API_FAVORITE_ROUTE,
+  API_FAVORITES_ROUTE,
+} from 'routes';
+
+export const createFavoriteService = (token, body) => new Promise((resolve, reject) => {
```

Comment:
`apiAddFavorite`

The verb "create" is not wrong at all. However, this is not a service.